core/storage/btree.rs

```diff
@@ -247,6 +242,32 @@ pub enum OverwriteCellState {
     },
 }
 
+#[derive(Debug, Clone)]
```

Comment:
Why do we need Clone for it?

Reply:
We don't, good spot! There was a point in time where I thought we did need it, so it's a leftover